src/Symfony/Component/Notifier/Bridge/Twilio/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.4
+---
+
+ * Support WhatsApp integration, allowing `From` numbers the `whatsapp:` prefix
```

Comment:
I wonder if the same about the `whatsapp:` prefix also isn't true for the receiver

Reply:
I think I would reword the changelog a bit then:

```
 * Add support for the WhatsApp integration if phone numbers are prefixed with `whatsapp:`
```